src/library/basetracktablemodel.cpp

```diff
@@ -613,6 +618,7 @@ QVariant BaseTrackTableModel::roleValue(
             return QString("(%1)").arg(timesPlayed);
         }
         case ColumnCache::COLUMN_LIBRARYTABLE_DATETIMEADDED:
+        case ColumnCache::COLUMN_LIBRARYTABLE_LAST_PLAYED_AT:
```

Comment:
COLUMN_LIBRARYTABLE_LAST_PLAYED_AT is not UTC. We need an extra case here.

Reply:
@daschuer It actually is stored as UTC, so this code correct. Please double check!

Reply:
No, not with current upstream/main. 

If we decide for UTC in the library, the code her is correct.

Reply:
We cannot decide about this if the system already decided for us. At least for my setup the timestamps are stored in UTC (see comments).